packages/app-runtime/test/modules/IdentityDeletionProcessStatusChangedModule.test.ts

```diff
@@ -22,23 +27,34 @@ describe("IdentityDeletionProcessStatusChanged", function () {
         if (activeIdentityDeletionProcess.value.status === IdentityDeletionProcessStatus.Approved) {
             const abortResult = await session.transportServices.identityDeletionProcesses.cancelIdentityDeletionProcess();
             if (abortResult.isError) throw abortResult.error;
+
+            if (sessionDevice2 && runtimeDevice2 && sessionDevice2.account.deletionDate) {
+                await sessionDevice2.transportServices.account.syncDatawallet();
+                await eventBus.waitForRunningEventHandlers();
+            }
         }
     });
 
-    afterAll(async () => await runtime.stop());
+    afterAll(async function () {
+        await runtime.stop();
+        await runtimeDevice2?.stop();
 
-    test("should set the deletionDate on the LocalAccount initiating an IdentityDeletionProcess", async function () {
+        // this eventbus is not defenitely cleaned up with runtime2, so it needs to be closed manually
```

Comment:
`runtime2` isn't declared as variable in this test file, please write `runtimeDevice2` instead or rename the variable.